zebra/zebra_evpn_mh.c

```diff
@@ -325,16 +325,26 @@ zebra_evpn_es_evi_show_entry_detail(struct vty *vty,
 	if (json_array) {
 		json_object *json;
 		json_object *json_flags;
+                json_object *json_types;
 
 		/* Separate JSON object for each es-evi entry */
 		json = json_object_new_object();
 
 		json_object_string_add(json, "esi", es_evi->es->esi_str);
 		json_object_int_add(json, "vni", es_evi->zevpn->vni);
+                if (es_evi->flags & ZEBRA_EVPNES_EVI_LOCAL) {
+                        json_types = json_object_new_array();
+                        if (es_evi->flags & ZEBRA_EVPNES_EVI_LOCAL)
```

Comment:
Why do we need to check twice for the same `es_evi->flags & ZEBRA_EVPNES_EVI_LOCAL`?

Reply:
Done